### PR TITLE
fix: update link

### PR DESCRIPTION
**Pull request by ghost**
# Description
Updating link in communication guide to include writing style guide

## Type of change
- [x] Documentation (update or new)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: docs/communication/index.md [hunk 1]
```diff
@@ -27,4 +27,4 @@ For more information:
 - [Fearless feedback](fearless-feedback.md)
 - [Shoutouts](fearless-feedback.md#shoutouts)
 - [Scenarios](scenarios.md)
-- Writing Style Guide
+- [Writing Style Guide](docs/../../processes/documentation/writing-style-guide.md)
```
Comment (tancnle) [inline docs/communication/index.md]: **suggestion:** Could we use a relative link?

```suggestion
- [Writing Style Guide](../processes/documentation/writing-style-guide.md)
```

Comment (ghost) [reply docs/communication/index.md]: Can do!

**Review by tancnle — APPROVED** [review]
Boom 💥 Let's launch 🚀 Thanks, @talzid.

**Review by Ishrat420 — COMMENTED** [review]
Good job keeping the documentation up to date. This is informative !

**Pull request merged**